api/agent/server.go

```diff
@@ -589,13 +589,13 @@ func (s *Server) GetRouter() http.Handler {
 	// Spire APIs with versioning
 	apiRtr.HandleFunc("/api/v1/spire/debugserver", s.debugServer)
 	apiRtr.HandleFunc("/api/v1/spire/healthcheck", s.healthcheck)
```

Comment:
Can we also add methods here? Let's change `debugserver` to `serverinfo` and make it a `GET` method. 
Healthcheck can also be a GET method